### PR TITLE
Added Spiralling Madness Test

### PR DESCRIPTION
**Pull request by bcpettifer**
“Zaphod felt he was teetering on the edge of madness and wondered if he shouldn't just jump over and have done with it.”

spiral1

### CODE REVIEW

**Comment by ataulm** [comment]
Zb approves, nice one Ben!

**Comment by ryanbateman** [comment]
woaaah

**Comment by blundell** [comment]
:+1: :dancers:

File: mobile/src/androidTest/java/com/novoda/canvas/SpirallingMadnessTest.java [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+            Bitmap circle = createCircleBitmap(COLOURS[i % COLOURS.length], d);
+            imageView.setImageBitmap(circle);
+            imageView.setLayoutParams(new ActionBar.LayoutParams(d, d));
+            float xOffset = (i == 0) ? d/2 : (d/2) * OFFSET_MULTIPLIER;
+            float yOffset = (i == 0)? d/2 : (d/2) * OFFSET_MULTIPLIER;
+            imageView.setX(centreX - xOffset);
+            imageView.setY(centreY - yOffset);
+            circles.add(imageView);
+
+            d -= d / circlesCount;
```
Comment (blundell) [inline mobile/src/androidTest/java/com/novoda/canvas/SpirallingMadnessTest.java:79]: i like dis

**Pull request merged**